### ISSUE TITLE
Incompatible with BOSH cli v2 tarballs

### ISSUE THREAD

**Issue by WarheadsSE** [cf-platform-eng/tile-generator#183]
https://github.com/cf-platform-eng/tile-generator/blob/master/tile_generator/bosh.py#L67

This line hard-locks for the tar file to contain `./release.MF`, which is no longer a member of a release tarball provided by BOSH Cli v2. Now, it has no trailing dot: `release.MF`. This prevents building tiles from bosh releases made with BOSH Cli v2.

For now, I have patched my docker container in this fashion so that I can continue:
```patch
--- /usr/lib/python2.7/site-packages/tile_generator/bosh.py                                 
+++ /usr/lib/python2.7/site-packages/tile_generator/bosh.py
@@ -64,7 +64,7 @@
 
        def get_manifest(self, tarball):
                with tarfile.open(tarball) as tar:
-                       manifest_file = tar.extractfile('./release.MF')
+                       manifest_file = tar.extractfile('release.MF')
                        manifest = yaml.safe_load(manifest_file)
                        manifest_file.close()
                        return manifest
```

**Comment by mboldt** [comment]
We had some delay in our CI environment, but this fix is making its way through. Thank you, @WarheadsSE!

**Comment by mboldt** [comment]
Fix in tile-generator 5.0.7, now published.
